app/signals/apps/classification/models/training_set.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.gis.db import models
+
+from signals import settings
+from signals.apps.services.domain.checker_factories import ContentCheckerFactory
+from signals.apps.services.domain.mimetypes import MimeTypeFromContentResolverFactory, \
+    MimeTypeFromFilenameResolverFactory
+from signals.apps.services.validator.file import MimeTypeAllowedValidator, MimeTypeIntegrityValidator, \
+    ContentIntegrityValidator, FileSizeValidator
+
+
+class TrainingSet(models.Model):
+    """
+    This model represents a training set consisting of a single XLSX file, with a "Main", "Middle", "Sub" and "Text" column
```

Comment:
Ik zou naar een structuur toegaan die overeen komt met het model in Signalen:

- main
- sub
- text